plans/issue-1130-fsm-modal-reload.md

```
@@ -0,0 +1,266 @@
+
+## Concrete Steps
+
+Run these commands from `/Users/christian/Developer/quickadd`.
+
+1. Install FSM dependency.
+
+    bun add xstate
+
+2. Implement shared controller and tests.
+
+    # edit src/gui/utils/modalReloadMachine.ts
```

markdownlint-cli2 (0.21.0)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)